### PR TITLE
Template Pipeline: commenting and naming cleanups

### PR DESCRIPTION
**Pull request by dylhunn**
No new features -- just a variety of small commenting and naming cleanups to improve code health. See individual commits for details.

### CODE REVIEW

File: packages/compiler/src/template/pipeline/ir/src/ops/update.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ import {ParseSourceSpan} from '../../../../../parse_util';
 import {BindingKind, I18nParamResolutionTime, OpKind} from '../enums';
 import type {ConditionalCaseExpr} from '../expression';
 import {Op, XrefId} from '../operations';
-import {ConsumesSlotOpTrait, ConsumesVarsTrait, DependsOnSlotContextOpTrait, TRAIT_CONSUMES_VARS, TRAIT_DEPENDS_ON_SLOT_CONTEXT} from '../traits';
```
Comment (mmalerba) [inline packages/compiler/src/template/pipeline/ir/src/ops/update.ts]: If we want this to stop happening we can enable the following setting in `settings.json` (I have this in my user settings already):

```json
  "editor.codeActionsOnSave": {
    "source.organizeImports": true
  }
```

**Review by mmalerba — APPROVED** [review]

File: packages/compiler/src/template/pipeline/src/phases/empty_elements.ts [hunk 2]
```diff
@@ -24,7 +24,7 @@ const IGNORED_OP_KINDS = new Set([ir.OpKind.Pipe]);
  * Replace sequences of mergable elements (e.g. `ElementStart` and `ElementEnd`) with a consolidated
  * element (e.g. `Element`).
  */
-export function phaseEmptyElements(job: CompilationJob): void {
+export function collapseEmptyElements(job: CompilationJob): void {
```
[the remark this answers is not in the document]
Comment (dylhunn) [reply packages/compiler/src/template/pipeline/src/phases/empty_elements.ts:27]: Blocks sounds like it refers to the new control flow syntax -- how about `collapseEmptyInstructions` instead?

File: packages/compiler/src/template/pipeline/ir/src/ops/update.ts (hunk 1 above)
Comment (dylhunn) [reply packages/compiler/src/template/pipeline/ir/src/ops/update.ts]: Cool, I changed my settings, hopefully this solves it going forward

**Comment by dylhunn** [comment]
caretaker: Saucelabs appears to be a flake. No impact on g3.

File: packages/compiler/src/template/pipeline/src/phases/empty_elements.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (mmalerba) [reply packages/compiler/src/template/pipeline/src/phases/empty_elements.ts]: works for me

**Comment by alxhub** [comment]
This PR was merged into the repository by commit 184d8246cbe3432347a374dc46c2d3af4955e113.

**Pull request closed without merging**